src/main/java/underdogs/devbie/answer/dto/AnswerCreateRequest.java

```diff
@@ -0,0 +1,31 @@
+package underdogs.devbie.answer.dto;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import underdogs.devbie.answer.domain.Answer;
+import underdogs.devbie.answer.domain.AnswerContent;
+
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor
```

Comment:
@Builder를 사용하고 아래 처럼 접근제한자를 변경하는 것은 어떨까요?
```suggestion
@AllArgsConstructor(access = AccessLevel.PRIVATE)
```

Reply:
Builder 패턴을 사용하는 이유는 **점층적인 생성자 패턴의 안정성과 자바빈 패턴의 가독성** 라고 합니다. 

그래서 이펙티브 자바에서는 '필드의 수가 4개 이상일 때 빌더 패턴을 사용하는 것이 효율적이다.'라고 하더라구요!

위 상황에서는 `of` 정적 팩토리 메서드를 사용하는 것이 조금 더 이상적이지 않을까 하는 생각이 듭니다! :)

조언 감사합니다! :)

[출처](https://madplay.github.io/post/builder-when-faced-with-many-constructor-parameters)